lnst/Recipes/ENRT/LinuxBridgeOverBondRecipe.py

```diff
@@ -166,5 +173,28 @@ def generate_perf_endpoints(self, config: EnrtConfiguration) -> list[Collection[
         return [ip_endpoint_pairs(config, (self.matched.host1.br0, self.matched.host2.br0))]
 
     @property
-    def mtu_hw_config_dev_list(self):
+    def pause_frames_dev_list(self):
         return [self.matched.host1.eth0, self.matched.host2.eth0]
+
+    @property
+    def offload_nics(self):
+        return [self.matched.host1.bond0, self.matched.host2.bond0]
```

Comment:
should offloads be configured on the bond nic? shouldn't this also be `host1.eth0, etc...`?

Reply:
Configuration on bond interfaces works fine. This is intentional. Similar to what we have in BondRecipe and DoubleBondRecipe. The offload configuration is automatically applied to underlying ethernet devices.

Reply:
ok resolving and approving this PR.